controllers/submariner/broker_controller.go

```diff
@@ -63,6 +63,10 @@ func (r *BrokerReconciler) Reconcile(request ctrl.Request) (ctrl.Result, error)
 		// Error reading the object - requeue the request.
 		return reconcile.Result{}, err
 	}
+	if instance.ObjectMeta.DeletionTimestamp != nil {
+		// Graceful deletion has been requested, ignore the object
+		return reconcile.Result{}, nil
+	}
```

Comment:
nit - newlines for better readability.

```suggestion

	if instance.ObjectMeta.DeletionTimestamp != nil {
		// Graceful deletion has been requested, ignore the object
		return reconcile.Result{}, nil
	}
	
```